ob_v3p0/impl/extensions.md

```diff
@@ -0,0 +1,1353 @@
+        "issuer": {
+            "id": "https://example.com/issuers/876543",
+            "type": "Profile",
+            "name": "Example Corp"
+        },
+        "validFrom": "2010-01-01T00:00:00Z",
+        "name": "Teamwork Badge",
+        "credentialSubject": {
+            "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+            "type": "AchievementSubject",
+            "achievement": {
+                "id": "https://example.com/achievements/21st-century-skills/teamwork",
+                "type": ["Achievement", "Place"],
+                "criteria": {
+                    "narrative": "Team members are nominated for this badge by their peers and recognized upon review by Example Corp management."
+                },
+                "description": "This badge recognizes the development of the capacity to collaborate within a group environment.",
+                "name": "Teamwork",
+                "place": {
+                    "name": "Stadium of Light, Sunderland",
+                    "description": "A foodball stadium in Sunderland, England that is home to Sunderland A.F.C.",
+                    "geo": {
+                        "latitude": 54.914440,
+                        "longitude": -1.387721
+                    }
+                }
+            }
+        },
+        "credentialSchema": [{
+            "id": "https://purl.imsglobal.org/spec/ob/v3p0/schema/json/ob_v3p0_achievementcredential_schema.json",
+            "type": "1EdTechJsonSchemaValidator2019"
+        }, {
+            "id": "https://openbadgespec.org/extensions/geoCoordinatesExtension/schema_achievement_credential_obv3p0.json",
+            "type": "1EdTechJsonSchemaValidator2019"
+        }, {
+            "id": "https://openbadgespec.org/extensions/geoCoordinatesExtension/schema_achievement_obv3p0.json",
+            "type": "1EdTechJsonSchemaValidator2019"
+        }],
+        "place": {
+            "name": "Stadium of Light, Sunderland",
+            "description": "A foodball stadium in Sunderland, England that is home to Sunderland A.F.C.",
+            "geo": {
+                "latitude": 54.914440,
+                "longitude": -1.387721
+            }
+        }
+    }
+    </pre>
+
+- [Accessibility](https://www.imsglobal.org/sites/default/files/Badges/OBv2p0Final/extensions/index.html#-accessibility): This extension allows the addition of the content for people with disabilities.
+
+    This extension is not deprecated by the new version of the specs.
+    The adaptation of this extension ultimately consist on extending the
+    datamodel, adding existing accessibility attributes to the entities
+    `Achievement`, `AchievementCredential` and `Profile`.
+
+    The OB 3.0 JSON-LD context has already defined some terms that this
+    extension to all entities of the specification. So, you'll need a JSON-LD
+    context with the remaining terms.
+
+    <pre class="json example"
+        title="Accessibility OB 3.0 JSON-LD Context"
```

Comment:
Same comments apply to all the extensions, it seems better to just publish this new context somewhere instead of giving people instructions on how they can publish exactly this context somewhere themselves.